pallets/nfts/src/lib.rs

```diff
@@ -599,13 +636,20 @@ pub mod pallet {
 			attribute: PalletAttributes<T::CollectionId>,
 			value: BoundedVec<u8, T::ValueLimit>,
 		},
+		/// Multiple approvals of a collection or item were cancelled.
+		ApprovalsCancelled {
+			collection: T::CollectionId,
+			item: Option<T::ItemId>,
```

Comment:
Resolved by updating based on your suggestion. I don't think adding new field in the event later in the pallet if there are any logic made to the future pallet is complicated. So agree that this can be refactored to only use for the mentioned cases.